src/online-members/script.js

```diff
@@ -162,13 +162,16 @@ async function generateMembersList() {
 }
 
 function addEventListenerToMembersList() {
-  const membersList = document.querySelector(`#${MEMBERS_CONTAINER_ID} > ul`);
+  const membersList = document.querySelector(`#${MEMBERS_CONTAINER_ID} > UL`);
   membersList.addEventListener('click', (event) => {
-    const membersDiv = event.target.nodeName === 'DIV';
-    if (!membersDiv) {
+    const memberElement = event.target.closest(`.${MEMBERS_CLASS}`);
+    if (!memberElement) {
       return;
     }
```

Comment:
Is there any chance of dataset being null or undefined?

Reply:
Yes is element which does not have data set is selected then it will be null